cbor/suit_uri.diag.txt

```diff
@@ -0,0 +1,50 @@
+/ SUIT_Envelope_Tagged / 107( {
```

Comment:
For consistency with the normative part of the spec, this should use SUIT_Envelope, not SUIT_Envelope_Tagged.
See the active discussion on issue 262 and PR 268.